src/web/app/pages-logs/logs-page.component.spec.ts

```diff
@@ -98,10 +161,94 @@ describe('LogsPageComponent', () => {
     expect(logSpy).toHaveBeenCalledWith({
       searchFrom: '0',
       searchUntil: '0',
-      severities: 'INFO',
+      logEvent: 'REQUEST_RECEIVED',
+      advancedFilters: {
+        traceId: 'testTrace',
+        googleId: 'testGoogleId',
+        sourceLocationFile: 'testFile',
+        sourceLocationFunction: 'testFunction',
+      },
     });
+  });
+
+  it('should display error messgae if severity level is not selected', () => {
```

Comment:
Typo in `message`; happens a few times in this file

Reply:
Fixed!